### PR TITLE
Add incomplete invoices feature

### PR DESCRIPTION
**Pull request by mballantyne3**
Added feature incomplete invoices with associated specs. Also added class method to `InvoiceItem` class to be used in the controller for the views.

### CODE REVIEW

File: spec/features/admin/index_spec.rb [hunk 1]
```diff
@@ -113,6 +113,22 @@
...
+          invoice_7 = Invoice.create!(status: 0, customer_id: @customer_1.id)
+          invoice_item_7 = InvoiceItem.create!(quantity: 4, unit_price: 434, status: 0, item_id: item_7.id, invoice_id: invoice_7.id)
+
+          visit "/admin"
+
+          within("#incompleteInvoices") do
+          expect(page).to have_content("Incomplete Invoices")
+          expect(page).to have_content("Invoice ID number: #{invoice_7.id}")
+          expect(page).to_not have_content("Invoice ID number: #{@invoice_6.id}")
+          end
```
Comment (RyanChrisSmith) [inline spec/features/admin/index_spec.rb:129]: great use of edge case testing!

**Review by RyanChrisSmith — APPROVED** [review]
Looks great! Thanks for moving all those test files around!
